docs/en/ingest-management/integrations/integrations.asciidoc

```diff
@@ -61,3 +61,4 @@ The *Integrations* app in {kib} needs access to the public {package-registry} to
 discover integrations. If your deployment has network restrictions, you can
 {fleet-guide}/air-gapped.html#air-gapped-diy-epr[deploy your own self-managed {package-registry}].
 ====
+
```

Comment:
I'd maybe remove this extra line just to avoid including an extra file in the PR.